src/http/v2/ngx_http_v2_filter_module.c

```diff
@@ -465,10 +470,17 @@ ngx_http_v2_header_filter(ngx_http_request_t *r)
 
     if (r->headers_out.server == NULL) {
 
+#if (T_NGX_SERVER_INFO)
```

Comment:
not recommend to fix these debug lines, too many to manage diff, see following:


![image](https://user-images.githubusercontent.com/623062/54995574-f78e5380-5001-11e9-8757-aee1aca70ea2.png)